src/components/PageNameHeader/index.js

```diff
@@ -8,4 +10,8 @@ const PageNameHeader = ({ pageName }) => {
   );
 };
 
+PageNameHeader.propTypes = {
```

Comment:
What is this doing:)

Reply:
See [this documentation](https://reactjs.org/docs/typechecking-with-proptypes.html)!